version.gradle.kts

```diff
@@ -1 +1 @@
-val PROSPECTIVE_RELEASE_VERSION by extra("2.2.0")
+val PROSPECTIVE_RELEASE_VERSION by extra("2.3.0")
```

Comment:
2.2.0 still hasn't been released! https://github.com/VirtusLab/git-machete-intellij-plugin/releases -> 2.1.0 is the latest

Reply:
So 2.2.0 should stay as is